lib/ret_web/controllers/api/v1/oauth_controller.ex

```diff
@@ -121,9 +157,9 @@ defmodule RetWeb.Api.V1.OAuthController do
     end
   end
 
-  # If an oauthprovider exists for the given discord_user_id, return the associated account, updating the email
+  # If an oauthprovider exists for the given discord_user_id (chat_user_id), return the associated account, updating the email
```

Comment:
Might as well stick with chat_user_id, since we've removed all references to discord_user_id.
```suggestion
  # If an oauthprovider exists for the given chat_user_id, return the associated account, updating the email
```